libert_util/tests/ert_util_sscan_test.c

```diff
@@ -65,7 +65,33 @@ void test_sscan_percent() {
 }
 
 
+void test_date(time_t expected , const char * date_string, bool expected_return) {
+  time_t t;
+  bool valid = util_sscanf_isodate(date_string, &t);
+
+  test_assert_bool_equal( valid , expected_return );
+  if (valid)
+    test_assert_time_t_equal( t , expected );
+  else
+    test_assert_time_t_equal( t , -1 );
+}
+
+
+void test_scan_iso_date() {
+  time_t expected = util_make_date_utc(10,  11 , 2011);
+
+  test_date( expected , "2011-11-10", true);
+
+  test_assert_false( util_sscanf_isodate( "2017.10.07" , NULL ));
+  test_assert_false( util_sscanf_isodate( "2017-10.7" , NULL ));
+  test_assert_false( util_sscanf_isodate( "2017/10/07" , NULL ));
+
```

Comment:
Maybe also test that `1995-13-01` is not a date, but `1993-12-30` is a date?  So that we know that months are months and days are days?